.env.template

```diff
@@ -131,18 +131,25 @@ GDS_BFF_LOG_LEVEL=info
 GDS_BFF_CONSOLE_LOG=true
 GDS_BFF_ALLOW_ORIGINS="http://localhost:3000,http://localhost:3003"
 GDS_BFF_COOKIE_DOMAIN=localhost
+
+GDS_BFF_AUTH0_ISSUER=
+GDS_BFF_AUTH0_AUDIENCE=
+
 GDS_BFF_TESTNET_INSECURE=true
 GDS_BFF_TESTNET_ENDPOINT=localhost:4436
 GDS_BFF_TESTNET_TIMEOUT=5s
+
 GDS_BFF_MAINNET_INSECURE=true
 GDS_BFF_MAINNET_ENDPOINT=localhost:8436
 GDS_BFF_MAINNET_TIMEOUT=5s
+
 GDS_BFF_DATABASE_URL=trtl://localhost:4436/
 GDS_BFF_DATABASE_REINDEX_ON_BOOT=false
 GDS_BFF_DATABASE_INSECURE=true
 GDS_BFF_DATABASE_CERT_PATH=
 GDS_BFF_DATABASE_POOL_PATH=
-GDS_BFF_TRACK_PERFORMANCE=false
+
+GDS_BFF_SENTRY_TRACK_PERFORMANCE=false
```

Comment:
Trying to keep the `.env.template` up to date with configuration changes -- organizing it a bit here too.